atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/sweep/AbstractSweepTaskRunnerTest.java

```diff
@@ -243,6 +244,7 @@ public void testSweeperFailsHalfwayThroughOnDeleteTable() {
                         .maxCellTsPairsToExamine(DEFAULT_BATCH_SIZE)
                         .build(),
                 nextStartRow);
+        Thread.sleep(1);
```

Comment:
Not sure I understand why we need the `Thread.sleep(1)` here - I get that it would be a failing test for the original impl with a non-Auxiliary start time for sweep, though I think we have the test in `SweepResultsTest` that already does that.

Reply:
Yep, that's why I put it there - I can revert that bit.

Reply:
Looks to me like it was left from debugging. I'd remove it